src/java.ts

```diff
@@ -132,12 +129,7 @@ export async function fetchJRE(
 
     await extractArchive(archivePath, jreDirPath);
 
-    const jreBinPath = path.join(jreDirPath, latestJREData.javaPath);
-
-    return {
-      ...latestJREData,
-      jrePath: jreBinPath,
-    };
```

Comment:
Other info arn't needed elsewhere, so I've changed this to just return the path to the binary

Reply:
thanks for the callout